From: =?utf-8?q?S=C3=A9bastien_Delafond?= <sdelafond@gmail.com>
Date: Sun, 5 Aug 2018 09:30:37 +0200
Subject: Do no run full coverage tests

---
 setup.cfg        | 81 --------------------------------------------------------
 test/conftest.py |  2 --
 2 files changed, 83 deletions(-)

diff --git a/setup.cfg b/setup.cfg
index 32b6aac..dc07038 100644
--- a/setup.cfg
+++ b/setup.cfg
@@ -9,86 +9,5 @@ addons = file,open,basestring,xrange,unicode,long,cmp
 testpaths = test
 addopts = --capture=no --color=yes
 
-[coverage:run]
-branch = False
-omit = *contrib*, *tnetstring*, *platform*, *main.py
-
-[coverage:report]
-show_missing = True
-exclude_lines =
-    pragma: no cover
-    raise NotImplementedError()
-
 [mypy-mitmproxy.contrib.*]
 ignore_errors = True
-
-[tool:full_coverage]
-exclude =
-    mitmproxy/proxy/protocol/base.py
-    mitmproxy/proxy/protocol/http.py
-    mitmproxy/proxy/protocol/http1.py
-    mitmproxy/proxy/protocol/http2.py
-    mitmproxy/proxy/protocol/http_replay.py
-    mitmproxy/proxy/protocol/rawtcp.py
-    mitmproxy/proxy/protocol/tls.py
-    mitmproxy/proxy/root_context.py
-    mitmproxy/proxy/server.py
-    mitmproxy/tools/
-    pathod/pathoc.py
-    pathod/pathod.py
-    pathod/test.py
-    pathod/protocols/http2.py
-    release/hooks
-
-
-[tool:individual_coverage]
-exclude =
-    mitmproxy/addons/onboardingapp/app.py
-    mitmproxy/addons/termlog.py
-    mitmproxy/contentviews/base.py
-    mitmproxy/controller.py
-    mitmproxy/ctx.py
-    mitmproxy/exceptions.py
-    mitmproxy/flow.py
-    mitmproxy/io/io.py
-    mitmproxy/io/tnetstring.py
-    mitmproxy/log.py
-    mitmproxy/master.py
-    mitmproxy/net/check.py
-    mitmproxy/net/http/cookies.py
-    mitmproxy/net/http/headers.py
-    mitmproxy/net/http/message.py
-    mitmproxy/net/http/multipart.py
-    mitmproxy/net/http/url.py
-    mitmproxy/net/tcp.py
-    mitmproxy/net/tls.py
-    mitmproxy/options.py
-    mitmproxy/proxy/config.py
-    mitmproxy/proxy/modes/http_proxy.py
-    mitmproxy/proxy/modes/reverse_proxy.py
-    mitmproxy/proxy/modes/socks_proxy.py
-    mitmproxy/proxy/modes/transparent_proxy.py
-    mitmproxy/proxy/protocol/base.py
-    mitmproxy/proxy/protocol/http.py
-    mitmproxy/proxy/protocol/http1.py
-    mitmproxy/proxy/protocol/http2.py
-    mitmproxy/proxy/protocol/http_replay.py
-    mitmproxy/proxy/protocol/rawtcp.py
-    mitmproxy/proxy/protocol/tls.py
-    mitmproxy/proxy/root_context.py
-    mitmproxy/proxy/server.py
-    mitmproxy/utils/bits.py
-    pathod/language/actions.py
-    pathod/language/base.py
-    pathod/language/exceptions.py
-    pathod/language/generators.py
-    pathod/language/http.py
-    pathod/language/message.py
-    pathod/log.py
-    pathod/pathoc.py
-    pathod/pathod.py
-    pathod/protocols/http.py
-    pathod/protocols/http2.py
-    pathod/protocols/websockets.py
-    pathod/test.py
-    release/hooks
diff --git a/test/conftest.py b/test/conftest.py
index 7c7dec4..da401b4 100644
--- a/test/conftest.py
+++ b/test/conftest.py
@@ -5,8 +5,6 @@ from mitmproxy.utils import data
 
 import pytest
 
-pytest_plugins = ('test.full_coverage_plugin',)
-
 skip_windows = pytest.mark.skipif(
     os.name == "nt",
     reason='Skipping due to Windows'
