From: =?utf-8?q?S=C3=A9bastien_Delafond?= <sdelafond@gmail.com>
Date: Thu, 9 Nov 2017 16:31:45 +0100
Subject: Remove conflict on kaitaistruct < 0.7

---
 setup.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/setup.py b/setup.py
index 0079f77..d5baa59 100644
--- a/setup.py
+++ b/setup.py
@@ -70,7 +70,7 @@ setup(
         "html2text>=2016.1.8, <=2016.9.19",
         "hyperframe>=4.0.1, <5",
         "jsbeautifier>=1.6.3, <1.7",
-        "kaitaistruct>=0.6, <0.7",
+        "kaitaistruct>=0.6",
         "passlib>=1.6.5, <1.8",
         "pyasn1>=0.1.9, <0.3",
         "pyOpenSSL>=16.0, <17.0",
