From: =?utf-8?q?S=C3=A9bastien_Delafond?= <sdelafond@gmail.com>
Date: Thu, 9 Nov 2017 16:34:29 +0100
Subject: Remove conflict on hyperframe < 0.5

---
 setup.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/setup.py b/setup.py
index d5baa59..a0b6391 100644
--- a/setup.py
+++ b/setup.py
@@ -68,7 +68,7 @@ setup(
         "cssutils>=1.0.1, <1.1",
         "h2>=2.5.1, <3",
         "html2text>=2016.1.8, <=2016.9.19",
-        "hyperframe>=4.0.1, <5",
+        "hyperframe>=4.0.1",
         "jsbeautifier>=1.6.3, <1.7",
         "kaitaistruct>=0.6",
         "passlib>=1.6.5, <1.8",
